Gemfile

```diff
@@ -11,9 +11,10 @@ gemspec
 # your gem to rubygems.org.
 
 # *** NFG specific gems - Start *** #
-source "https://rubygems.pkg.github.com/network-for-good" do
-  # NFG gems for authenticating using the NFG openid server
-  gem "sso_openid", '>= 6.0.5'
+group :test, :development do
```

Comment:
Is this only a development dependency? Does this mean that in production this gem doesn't need the sso_openid gem?